framework/doc/content/source/postprocessors/ElementHDivSemiError.md

```diff
@@ -0,0 +1,31 @@
+# ElementHDivSemiError
+
+!syntax description /Postprocessors/ElementHDivSemiError
+
+## Overview
+
+The H(div)-semierror between a vector-valued solution $\vec{u}$ and a known
+vector-valued function $\vec{f}$ is written
+
+\begin{equation}
+    \left( \int_{\Omega} | \nabla \cdot \vec{u}  - \nabla \cdot \vec{f} |^2 \,\text{d}x \right) ^{1/2}
+\end{equation}
+
+where $|\cdot|$ denotes the absolute value.
+
+The nonlinear vector variable specified for $\vec{u}$ must be of the
+`RAVIART_THOMAS` family, as MOOSE currently does not compute the divergence for
+any other finite element family.
```

Comment:
I am changing this in #26477 so I would probably just remove this paragraph

Reply:
Correct me if I'm wrong.
You are changing things in #26477 so the decision to know if divergences are needed or not will depend simply on a request for that quantity by the user? That means, divergences will now be available for all vector fe families (except for `NEDELEC_ONE` for which the H(curl) Piola mapping is missing libMesh)?
I see you don't intend to make the same change for curls, so should I keep the disclaimer in `ElementHCurlSemiError.md`?

Reply:
Correct on all counts